pyrasterframes/src/main/python/docs/unsupervised-learning.pymd

```diff
@@ -89,7 +90,7 @@ model = pipeline.fit(df)
 
 We can use the `transform` function to score the training data in the fitted _pipeline_ model. This will add a column called `prediction` with the closest cluster identifier.
 
-```python, transform
+```python, transform, results='raw'
```

Comment:
here we also have an issue with a really wide table. may make sense just to select the features that best align to our message. `prediction` and perhaps some of the metadata?